canonical-websites/build.snapcraft.io
# 913 icon components

## Done

- Created the new Icons react component, this has pre-build icons and the ability to refine any new icons using `<Icon apperance='error' />` or e.g `<IconError />`.
- Replaced all icons on the repository pages to use the new spinner, success, error and copy icons.
- Removed Icon styles from Repository row
- Updated the copy share function to use the new `<IconCopy />`
- Updated the button component to use the new spinner

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- View any new icons at the repository listing, Error / Success. 
- Add a new repo to see the spinner icon working in a button
- Build copy icons have also been updated, check a failed build page to see the new icon.


## Issue / Card

Fixes #913
